scripts/kmodtool

```diff
@@ -144,6 +144,12 @@ print_rpmtemplate_per_kmodpkg ()
 	local kernel_uname_r=${1}
 	local kernel_variant="${2:+-${2}}"
 
+	# With RHEL 8, depmod moved from /sbin to /usr/sbin
+	local depmod_path=/sbin/depmod
+	if [ ! -f ${depmod_path} ]; then
+		depmod_path=/usr/sbin/depmod
```

Comment:
given that /sbin/depmod and /usr/sbin/depmod both symlinked to /usr/bin/kmod, is it true that there will always be /sbin/depmod file and depmod_path will never be /usr/sbin/depmod?

Reply:
Yes and no.  This is true for CentOS/RHEL 7 and 8, plus any semi-recent Fedora release.  It's not the case for CentOS/RHEL 6, which is ancient, but I'd really prefer not to break if at all possible.  I'm also unsure of the situation on other platforms so my feeling is a minimal change is desirable.

@mdiep25 by chance were you able to test this patch and verify it does resolve the packaging issue?

Reply:
yes, I have and it didn't fix the problem. rpm -ihv works but not yum/dnf. I verified it by make the Requires = kmod and it worked. see https://review.whamcloud.com/#/c/36095/5/contrib/lbuild/rhel8/zfs_issue_8724.patch

Reply:
That's a pretty nice way to handle it.  I think CentOS/RHEL 6 is the distro still providing depmod as part of the `module-init-tools` package, so what do you think of something like this in addition to what you've proposed.

```
%if 0%{?rhel} == 6 || 0%{?centos} == 6
Requires(post):   module-init-tools
Requires(postun): module-init-tools
%else
Requires(post):   kmod
Requires(postun): kmod
%endif
```

Reply:
Thanks, I was just trying to test it. yeah, your proposed change looks good to me